scripts/python_scripts/generate_network_status.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+
+import json
+import os
+from jinja2 import Template
+from pytablewriter import MarkdownTableWriter
+from subquery_cli import use_subquery_cli
+
+subquery_cli_version = '0.2.5'
```

Comment:
in PR description you wrote `was changed workflows for 0.2.6 version of subquery CLI.` but here you use `0.2.5` version. Is it okay?

Reply:
Yes, this is used for a special case where the improvements from 0.2.6 are not involved